### ISSUE TITLE
AppDelegate w/Storyboard implementation

### ISSUE THREAD

**Issue by cjazz** [MatthewYork/iPhone-IntroductionTutorial#4]
Hi Matt, I wouldn't call this an issue but I'm having difficulty implementing the launch of a few MyIntroductionPanel's on first launch in my app.  I've got a storyboard, and trapping for NSUserDefaults - has run before.. 

UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"iPhone" bundle:nil];
        UIViewController *vc = [storyboard instantiateViewControllerWithIdentifier:@"MyStoryboardID"];
        //
        [self presentViewController:vc animated:YES completion:nil];

Issues: No visible @interface for 'AppDeligate' declares the selector 'presentViewController:animated:completion'

I wouldn't call an issue of your code, but my lack of knowledge on implementation.  Just thought I'd post if you knew a quick path.. I'll search the web also, and post back.

Thanks for your great solution, it works beautifully everywhere in my first app (getting ready to submit)!

**Comment by MatthewYork** [comment]
Thanks for trying out the introductionView, cjazz! If i had any suggestions it would be to present the introduction in whatever view controller you consider the root or "home" view controller instead of the appdelegate. At that point, you should be able to use the same code listed in the github readme. Just implement it in whatever method you want that begins when the VC is loaded.

**Comment by cjazz** [comment]
Hi Matt, thanks for the speedy reply.  As I examined different approaches this morning, I realized the best way is as you suggested.  Using my starting view in my story board as a home base is what I did.   I've utilized introduction view at start, and on various main screens via 'info" button in navbar.  I'm getting ready to submit to the app store tonight.. Here's my web site for the app http://www.hushboxlive.com.    Thank you so much!
